Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -1448,12 +1448,37 @@ public static bool Install(NugetPackage package, bool refreshAssets = true)
                 {
                     EditorUtility.DisplayProgressBar(string.Format("Installing {0} {1}", package.Id, package.Version), "Importing Package", 0.95f);
                     AssetDatabase.Refresh();
+                    ModifyImportSettingsIfRoslynAnalyzer(package.Id, package.Version);
                     EditorUtility.ClearProgressBar();
                 }
             }
             return installSuccess;
         }
 
+        private static void ModifyImportSettingsIfRoslynAnalyzer(string id, string version)
+        {
+            var dir = $"Assets/Packages/{id}.{version}";
```

Comment:
```suggestion
            var dir = Path.Combine(NugetConfigFile.RepositoryPath, $"{id}.{version}");
            if (!Directory.Exists(dir))
            {
                return;
            }
```

Reply:
To get it working locally I made the following changes

```c#
var dir = Path.Combine(NugetConfigFile.RepositoryPath, $"{id}.{version}");
if (!Directory.Exists(dir))
{
    return;
}
foreach (var analyzer in Directory.GetFiles(dir, "*.dll", SearchOption.AllDirectories)
    .Where(x => x.Contains("/analyzers/")))
{
    // Trigger Unity to create the meta file, the path for ImportAsset MUST be relative to the project
    // path
    var projectRelativePath =
        analyzer.Substring(new DirectoryInfo(Application.dataPath).Parent.FullName.Length + 1);
    AssetDatabase.ImportAsset(projectRelativePath, ImportAssetOptions.ForceSynchronousImport);
    var meta = AssetImporter.GetAtPath(projectRelativePath) as PluginImporter;
```

Reply:
Fixed 9d60da5